tomoscan/tomoscan.py

```diff
@@ -860,8 +873,8 @@ def compute_frame_time(self):
         # Add some extra time to exposure time for margin.
         # Adding 1% to the exposure time, and at least 1 ms seems to work for FLIR cameras
         # This is empirical and should be made camera dependent
-        # VN: had to increae to 2.5% because Oryx had missing frames when working with 100fps
-        frame_time = exposure * 1.025        
+        # VN: had to increae to 3.5% because Oryx had missing frames when working with 100fps
```

Comment:
I suggest a change in how this is done, using my suggestion in the comment.  At the top of the function we add
``` 
       # Default readout time margin is 1%
       readout_margin = 1.01
````
Then in the camera specific code you can change this.  So move the lines you added into the Oryx specific code.

Reply:
changed

Reply:
There is still a # VN comment that should be moved or deleted here.

Reply:
Is removed. There was one more left [here](https://github.com/tomography/tomoscan/blob/9ddcae08d3e1659a19f832fcca5f3599b114e98d/tomoscan/tomoscan.py#L792) that I reference to the open [issue](https://github.com/tomography/tomoscan/issues/127).